pkg/pipelines/bootstrap.go

```diff
@@ -64,6 +68,13 @@ func Bootstrap(o *BootstrapOptions) error {
 			return errors.New("failed due to Tekton Pipelines or Triggers are not installed")
 		}
 	}
+
+	// Validate image repo
```

Comment:
These comments don't add anything to the code?

Reply:
Removed